KMD.Identity.TestApplications.OpenID.Angular/ClientApp/tsconfig.json

```diff
@@ -14,12 +14,13 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2017",
+    "target": "ES2022",
     "module": "es2020",
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "useDefineForClassFields": false
```

Comment:
I've just read about this useDefineForClassFields and it's a bit unclear for me. Can you say what will happen if this will be set to true?

Reply:
It was set automatically from the migration. From the docs:

   >This flag is used as part of migrating to the upcoming standard version of class fields. TypeScript introduced class fields many years before it was ratified in TC39. The latest version of the upcoming specification has a different runtime behavior to TypeScript’s implementation but the same syntax.

> This flag switches to the upcoming ECMA runtime behavior.

And more details: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#the-usedefineforclassfields-flag-and-the-declare-property-modifier

Perhaps we could skip setting it if we do not actually hit the case described, but maybe it is safer to just follow the migration.

Reply:
yeah, let's keep it as is, don't change anything :)